docs/literate/src/files/differentiable_programming.jl

```diff
@@ -146,27 +146,36 @@ The classical isentropic vortex test case of
   [NASA/CR-97-206253](https://ntrs.nasa.gov/citations/19980007543)
 """
 function initial_condition_isentropic_vortex(x, t, equations::CompressibleEulerEquations2D)
-  inicenter = SVector(0.0, 0.0) # size and strength of the vortex
-  iniamplitude = 0.2
-
-  rho = 1.0 # base flow
+  # for error convergence: make sure that the end time such, that the vortex is back at the initial state!!
+  # for the current vel and domain size: t_end should be a multiple of 20s
```

Comment:
```suggestion
  # for error convergence: make sure that the end time is such, that the vortex is back at the initial state!!
  # for the current velocity and domain size: t_end should be a multiple of 20s
```
Fix typo (here and elsewhere)?

Reply:
We could get around this restriction by implementing a time-dependent IC and using [`x_trans_periodic_2d`](https://github.com/trixi-framework/Trixi.jl/blob/2b9272c654f80e450998610ac719b95ba2947e2a/src/equations/linear_scalar_advection_2d.jl#L33-L39) to compute the appropriate coordinates. But I guess that this is beyond the scope of this PR (maybe for some student at a later time?).

Reply:
I can fix the typos in all the elixirs. Implementing the time-dependent IC could be useful. I remember @gregorgassner played around with this IC for about 15 minutes or so, trying to get such a coordinate shift working.

Reply:
Typos are fixed.